blocks-common/i-ua/i-ua.js

```diff
@@ -47,10 +71,26 @@
                 var _this = this,
                     self = _this.__self;
 
+                this.setMod('platform',
```

Comment:
Because it is already defined in this block.
On Чт, 9 апр. 2015 at 21:42, Alex Hohulin <notifications@github.com> wrote:

> In blocks-common/i-ua/i-ua.js
> <https://github.com/bem/bem-bl/pull/629#discussion_r28084429>:
>
> > @@ -47,10 +71,26 @@
> >                  var _this = this,
> >                      self = _this.__self;
> >
> > +                this.setMod('platform',
>
> But why? One context!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bem/bem-bl/pull/629/files#r28084429>.
>